scipy/stats/_continuous_distns.py

```diff
@@ -5206,6 +5206,321 @@ def _entropy(self, beta):
+        # because (B - x) ** (..) is also executed for x <= -alpha
+        # But these values are not used in the end, therefore we ignore the errors
+        with np.errstate(invalid='ignore', divide='ignore'):
+            return N * np.where(x > -alpha, np.exp(- x**2 / 2), A * (B - x) ** (-n))		
+    
+    def _cdf(self, x, alpha, n):
+        """
+        Return CDF of the crystalball function
+        """
+        A = (n / np.abs(alpha) ) ** n * np.exp( - alpha ** 2 / 2.0 )
+        B = n / np.abs(alpha)  - np.abs(alpha)
+        C = n / np.abs(alpha) * 1.0 / (n - 1.0) * np.exp( - alpha ** 2 / 2.0 )
+        D = np.sqrt(np.pi / 2.0) * (1 + sc.erf(np.abs(alpha) / np.sqrt(2) ))
+        N = 1.0 / (C + D)
+        # Using np.where we also calculate powers of negative numbers,
+        # because (B - x) ** (..) is also executed for x <= -alpha
+        # But these values are not used in the end, therefore we ignore the errors
+        with np.errstate(invalid='ignore', divide='ignore'):
+            return N * np.where(x > -alpha,
+                    A * (B + alpha) ** (-n+1) / (n-1) + _norm_pdf_C * (_norm_cdf(x) - _norm_cdf(-alpha)),
+                    A * (B - x) ** (-n+1) / (n-1) )
+    
+    def _argcheck(self, alpha, n):
+        """
+        In HEP crystal-ball is also defined for n = 1 (see plot on wikipedia)
+        But the function doesn't have a finite integral in this corner case,
+        and isn't a PDF anymore (but can still be used on a finite range).
+        Here we restrict the function to n > 1.
+        """
+        return (n > 1)
+crystalball = crystalball_gen(name='crystalball', longname="A Crystalball Function")
+
+
+def _argus_phi(chi):
+    """
+    Utility function for the argus distribution
+    used in the CDF and norm of the Argus Funktion
+    """
+    return  _norm_cdf(chi) - chi * _norm_pdf(chi) - 0.5
+
+
+class argus_gen(rv_continuous):
+    """
+    Argus distribution
+
+    %(before_notes)s
+
+    Notes
+    -----
+    Named after the argus experiment.
+    Used in elementary particle physics to model invariant mass distributions from continuum background
+
+    For details see: https://en.wikipedia.org/wiki/ARGUS_distribution
+
+    The parameter c of the Argus distributions is named scale in scipy.
+
+    %(after_notes)s
+
+    %(example)s
+    """
+    def __init__(self, *args, **kwargs):
+        """
+        Set finite support for argus function.
+        """
+        # Set support
+        kwargs['a'] = 0.0
+        kwargs['b'] = 1.0
+        super(argus_gen, self).__init__(*args, **kwargs)
+
+    def _pdf(self, x, chi):
+        """
+        Return PDF of the argus function
+        """
+        return chi**3 / (_norm_pdf_C * _argus_phi(chi)) * x * np.sqrt(1.0 - x**2) * np.exp(- 0.5 * chi**2 * (1.0 - x**2) )
+
+    def _cdf(self, x, chi):
+        """
+        Return CDF of the argus function
+        """
+        return 1.0 - _argus_phi(chi * np.sqrt(1 - x**2)) / _argus_phi(chi)
```

Comment:
ÒK, yes indeed it does. Then I'd add `def _cdf(...): return 1.0 - self._sf(...)`.